<ui version="4.0" >
 <class>SMFSettingsClass</class>
 <widget class="QWidget" name="SMFSettings" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>198</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>SMFSettings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Plugins</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="PluginlistboxBox" >
       <item>
        <property name="text" >
         <string>Facebook</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Flicker</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Orkut</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="pluginstatusbox" >
       <item>
        <property name="text" >
         <string>Enable</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Disable</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Auth. Exp. Limit</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="AuthExplimitbox" >
     <item>
      <property name="text" >
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>8</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>9</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>10</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Max. Data Transfer Limit</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="MaxDataTransferlimitBox" >
     <item>
      <property name="text" >
       <string>100</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>200</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>Network access in roaming</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="RoamingStatusBox" >
     <item>
      <property name="text" >
       <string>Enable</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Disable</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5" >
     <property name="text" >
      <string>Upload File Type</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="Uploadfiletypebox" >
     <item>
      <property name="text" >
       <string>Video</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Image</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Audio</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
